Formula/nest.rb

```diff
@@ -0,0 +1,152 @@
+
+  depends_on "libtool" => :run
+  depends_on "readline" => :run
+  depends_on "cmake" => :build
+
+  resource "Cython" do
+    url "https://files.pythonhosted.org/packages/ee/2a/c4d2cdd19c84c32d978d18e9355d1ba9982a383de87d0fcb5928553d37f4/Cython-0.27.3.tar.gz"
+    sha256 "6a00512de1f2e3ce66ba35c5420babaef1fe2d9c43a8faab4080b0dbcc26bc64"
+  end
+
+  resource "nose" do
+    url "https://files.pythonhosted.org/packages/58/a5/0dc93c3ec33f4e281849523a5a913fa1eea9a3068acfa754d44d88107a44/nose-1.3.7.tar.gz"
+    sha256 "f1bffef9cbc82628f6e7d7b40d7e255aefaa1adb6a1b1d26c69a8b79e6208a98"
+  end
+
+  fails_with :clang do
+    cause <<-EOS.undent
+      Building NEST with clang is not stable. See https://github.com/nest/nest-simulator/issues/74 .
+    EOS
+  end
+
+  def install
+    ENV.delete("CFLAGS")
+    ENV.delete("CXXFLAGS")
+    # The Homebrew site-packages path is automatically added to the PYTHONPATH
+    # env var in Library/Homebrew/requirements/python_requirement.rb. However,
+    # it is getting confused as to which version of Python we are using and
+    # putting the wrong site-packages on the path (2 instead of 3). Since we
+    # don't need any Homebrew-installed bindings it is easiest/safe just to
+    # delete it.
+    ENV.delete("PYTHONPATH")
+
+    args = ["-DCMAKE_INSTALL_PREFIX:PATH=#{prefix}"]
+
+    args << "-Dwith-mpi=ON" if build.with? "mpi"
+    args << "-Dwith-openmp=OFF" if build.without? "openmp"
+    args << "-Dwith-gsl=OFF" if build.without? "gsl"
+
+    if build.with? "python3"
+      args << "-Dwith-python=3"
+      python_exec = "python3"
+    else
+      # default to python2 installation
+      # this always links to the system python during nest installation,
+      # which should not be a problem, as python2.7.10 (system py2 as of writing this)
+      # is compatible with brewed python2.7+
+      args << "-Dwith-python=ON"
+      python_exec = "python"
+    end
+
+    python_version = Language::Python.major_minor_version(python_exec)
+
+    resource("nose").stage do
+      system python_exec, *Language::Python.setup_install_args(libexec/"nose")
+      # only nose executable is interesting during testing - not adding a *.pth file
+    end
+
+    resource("Cython").stage do
+      system python_exec, *Language::Python.setup_install_args(buildpath/"cython")
+    end
+
+    # Add local build resource Cython residing in buildpath to paths
+    ENV.prepend_create_path "PATH", buildpath/"cython/bin"
+    ENV.prepend_create_path "PYTHONPATH", buildpath/"cython/lib/python#{python_version}/site-packages"
+
+    # "out of source" build
+    mkdir "build" do
+      system "cmake", "..", *args
+      system "make"
+      system "make", "install"
+    end
+
+    # Replace internally accessible gcc with externally accesible version
+    # in nest-config if required
+    inreplace bin/"nest-config",
+        %r{#{HOMEBREW_REPOSITORY}/Library/Homebrew/shims.*/super},
+        "#{HOMEBREW_PREFIX}/bin",
+        FALSE
```

Comment:
Not sure what the deal is with this `inreplace` but any use of the `false` argument is essentially a bug by definition.

Reply:
See https://github.com/Homebrew/homebrew-science/pull/6479 why we introduced the `inreplace` and https://github.com/Homebrew/homebrew-science/pull/6488 for the `FALSE` part.